Adds bends for HV cables.

Lamrr opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Adds bends for HV cables, replacing the awkward box thing and making HV cables winding through maintenance a little more smoother. Aforementioned box thing is kept for single cable nodes and for junctions where it looks a bit more suitable.

Sprites are original work, derivatives of existing cables.

**Screenshots**
<!-- If applicable, add screenshots to showcase your PR. If your PR is a visual change, add
screenshots or it's liable to be closed by maintainers. -->
![Screenshot from 2022-02-10 22-33-26](https://user-images.githubusercontent.com/96937466/153401127-924f611c-c954-4f17-8e6d-119008d29aba.png)

**Changelog**
<!--
Here you can fill out a changelog that will automatically be added to the game when your PR is merged
There are 4 icons for changelog entries: add, remove, tweak, fix. I trust you can figure out the rest.

You can put your name after the :cl: symbol to change the name that shows in the changelog (otherwise it takes your GitHub username)
Like so: :cl: PJB

Generally, only put things in changelogs that players actually care about. Stuff like "Refactored X system, no changes should be visible" shouldn't be on a changelog.

For writing actual entries, don't consider the entry type suffix (e.g. add) to be "part" of the sentence:
bad: - add: a new tool for engineers
good: - add: added a new tool for engineers
-->

:cl:
- tweak: High-voltage cables are now a little smoother.

Lamrr commented:
I removed changes to the JSON file as it was causing RSI validation failures and I'm not sure why. Maybe some could look at the [old file ](https://github.com/space-wizards/space-station-14/pull/6622/commits/d3953907f4c24867fa494d0fdd1048347bb4bedf) and see if I missed something obvious?

Either way the license is unchanged so it doesn't really matter.

Peptide90 commented:
> I removed changes to the JSON file as it was causing RSI validation failures and I'm not sure why. Maybe some could look at the [old file ](https://github.com/space-wizards/space-station-14/pull/6622/commits/d3953907f4c24867fa494d0fdd1048347bb4bedf) and see if I missed something obvious?
> 
> Either way the license is unchanged so it doesn't really matter.

Not sure about json formatting but it might not have liked the _.

Besides that, I'll not bore people with the physics but good luck getting a bend like that on a HV cable in a 1 metre square haha. Jokes aside, looks nice.

Lamrr commented:
> > I removed changes to the JSON file as it was causing RSI validation failures and I'm not sure why. Maybe some could look at the [old file ](https://github.com/space-wizards/space-station-14/pull/6622/commits/d3953907f4c24867fa494d0fdd1048347bb4bedf) and see if I missed something obvious?
> > Either way the license is unchanged so it doesn't really matter.
> 
> Not sure about json formatting but it might not have liked the _.
> 
> Besides that, I'll not bore people with the physics but good luck getting a bend like that on a HV cable in a 1 metre square haha. Jokes aside, looks nice.

Strange, it also didn't work when I removed them.

Peptide90 commented:
> > > I removed changes to the JSON file as it was causing RSI validation failures and I'm not sure why. Maybe some could look at the [old file ](https://github.com/space-wizards/space-station-14/pull/6622/commits/d3953907f4c24867fa494d0fdd1048347bb4bedf) and see if I missed something obvious?
> > > Either way the license is unchanged so it doesn't really matter.
> > 
> > 
> > Not sure about json formatting but it might not have liked the _.
> > Besides that, I'll not bore people with the physics but good luck getting a bend like that on a HV cable in a 1 metre square haha. Jokes aside, looks nice.
> 
> Strange, it also didn't work when I removed them.

Ah I had this issue, remove the comma after the name lines. You can also remove the empty line below them if you like. You only need the comma if you have another variable like direction or delay.

SweptWasTaken commented:
Looks excellent, great work!
